### PR TITLE
More modules to custom elements

### PR DESCRIPTION
**Pull request by yebrahim**
- Converting GapiManager and TemplateManager to custom elements.
- Renaming datalab-files to file-browser.

### CODE REVIEW

File: sources/web/datalab/polymer/components/datalab-app/datalab-app.html [hunk 1]
```diff
@@ -76,7 +76,7 @@
           <iron-pages id="pages" selected="{{page}}" attr-for-selected="name"
                       fallback-selection="files">
             <datalab-data class="page" name="data"></datalab-data>
-            <datalab-files class="page" name="files"></datalab-files>
+            <file-browser class="page" name="files"><file-browser>
```
Comment (jimmc) [inline sources/web/datalab/polymer/components/datalab-app/datalab-app.html]: Needs a slash in the closing tag

File: sources/web/datalab/polymer/test/table-details-test.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  </head>
+  <body>
+
+    <test-fixture id="table-details-fixture">
+      <template>
+        <table-details></table-details>
+      </template>
+    </test-fixture>
+
+    <script src="table-details-test.js"></script>
```
Comment (jimmc) [inline sources/web/datalab/polymer/test/table-details-test.html]: I don't see a table-details-test.js file, and table-details.test does not appear in index.html. I think either those two should be fixed, or we should just delete this file for now until we actually have those tests.

Comment (yebrahim) [inline sources/web/datalab/polymer/test/table-details-test.html]: This file was added by mistake. Removed.

File: sources/web/datalab/polymer/components/datalab-app/datalab-app.html (hunk 1 above)
Comment (yebrahim) [inline sources/web/datalab/polymer/components/datalab-app/datalab-app.html]: Done.

**Pull request merged**